src/contexts/auth-context.tsx

```
@@ -90,6 +97,85 @@ export function AuthProvider({ children }: { children: ReactNode }) {
+                    setTelemetryEnabled(true);
+                }
+            }
+        })();
+
+        return () => {
+            cancelled = true;
+        };
+    }, [userData?.userId]);
+
+    useEffect(() => {
+        if (telemetryEnabled == null) {
```

ESLint

[error] Expected '===' and instead saw '=='.

(eqeqeq)